src/Application.php

```diff
@@ -103,15 +104,18 @@ class Application implements
      * @param mixed $configuration
      * @param ServiceManager $serviceManager
      */
-    public function __construct($configuration, ServiceManager $serviceManager)
-    {
+    public function __construct(
+        $configuration,
+        ServiceManager $serviceManager,
+        EventManagerInterface $events,
```

Comment:
Putting a note here as threads got broken up by me answering via email.

This change was for a reason, which I stated in the PR summary: I ended up with a circular reference issue with the ApplicationFactory + EventManagerAware initializer that resulted in invalid EventManager state. The result was that the EM instance injected no longer had the default listeners injected, which broke the entire thing.

As I noted in another comment elsewhere: hardly anybody directly instantiates the `Application` instance. The skeleton application pulls it from the service manager. As such, we can call this change out in the migration documentation for the very small percentage of users who are manually instantiating.